packages/valory/skills/safe_deployment_abci/payloads.py

```diff
@@ -46,7 +46,10 @@ class RandomnessPayload(BaseTxPayload):
     def __init__(
         self, sender: str, round_id: int, randomness: str, **kwargs: Any
     ) -> None:
-        """Initialize an 'select_keeper' transaction payload.
+        """Initialize an 'randomness' transaction payload.
```

Comment:
Added for clarification as it's otherwise hard to understand why we have this second property